test/spec/spec.form.js

```diff
@@ -515,14 +514,144 @@ describe('Form Controller', function () {
+                form.options.forks = [{
+                    target: '/target-page',
+                    condition: function () {
+                        return true;
+                    }
+                }];
+                form.getNextStep(req, {}).should.contain('/target-page');
+            });
+
+            it('returns the original next target if the condition function is not met', function () {
+                form.options.forks = [{
+                    target: '/target-page',
+                    condition: function () {
+                        return false;
+                    }
+                }];
+                form.getNextStep(req, {}).should.equal('/next-page');
+            });
+
+        });
+
+        describe('with more than one fork', function () {
+
+            describe('when the fields are the same', function () {
+
+                beforeEach(function () {
+                    req.form = {values: {
+                        'example-radio': 'condition-met'
+                    }};
+                    form.options.forks = [{
+                        target: '/target-page',
+                        condition: {
+                            field: 'example-radio',
+                            value: 'condition-met'
+                        }
+                    }, {
+                        target: '/target-page-2',
+                        condition: {
+                            field: 'example-radio',
+                            value: 'condition-met'
+                        }
+                    }];
+                });
+
+                it('retuns the last forks\' target if each condition is met', function () {
+                    form.getNextStep(req, {}).should.contain('/target-page-2');
+                });
+
+            });
+
+            describe('when the fields are different', function () {
+
+                beforeEach(function () {
+                    form.options.forks = [{
+                        target: '/target-page',
+                        condition: {
+                            field: 'example-radio',
+                            value: 'conditionMet'
+                        }
+                    }, {
+                        target: '/target-page-2',
+                        condition: {
+                            field: 'example-email',
+                            value: 'conditionMet'
+                        }
+                    }];
+                });
+
+                it('returns the last forks\' target if each condition is met', function () {
+                    req.form = {values: {
+                        'example-radio': 'conditionMet',
+                        'example-email': 'conditionMet'
+                    }};
+                    form.getNextStep(req, {}).should.contain('/target-page-2');
+                });
+
+                it('returns the first forks\' target if the first condition is met', function () {
```

Comment:
Do we need this test? It seems a bit misleading, like the behaviour is different. In this case the first fork's target is the first in the array but is still the last match.